charts/judicial-system/values.prod.yaml

```diff
@@ -19,7 +19,7 @@ judicial-system-api:
     AUTH_AUDIENCE: 'rettarvorslugatt.island.is'
     BACKEND_URL: 'http://web-judicial-system-backend'
     CONTENTFUL_HOST: 'cdn.contentful.com'
-    HIDDEN_FEATURES: ''
```

Comment:
Consider also hiding on staging, so the hiding can be tested there.

Reply:
Note, that the yaml files should not be modified directly. The judicial-system-api.ts file in the infra folder of the judicial-system-api application should be modified and the yaml files generated from it.

Reply:
Addressed here: https://github.com/island-is/island.is/pull/7545/commits/847862f2d6978e1e5be581917b13cbf7628580b5